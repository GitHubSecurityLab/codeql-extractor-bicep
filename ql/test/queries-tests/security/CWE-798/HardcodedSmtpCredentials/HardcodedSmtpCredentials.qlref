security/CWE-798/HardcodedSmtpCredentials.ql
