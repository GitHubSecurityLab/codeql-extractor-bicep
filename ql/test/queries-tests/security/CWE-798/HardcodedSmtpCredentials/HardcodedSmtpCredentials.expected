| app.bicep:26:19:26:40 | String | Hardcoded SMTP password in Grafana configuration. Use Azure Key Vault references instead. |
